examples/subplots_axes_and_figures/axhspan_demo.py

```diff
@@ -0,0 +1,37 @@
+"""
+============
+Axhspan Demo
```

Comment:
Should be "axhspan" (lowercase).  Probably there are other cases of wrong uppercasing, haven't checked.